verifier/verifier.go

```diff
@@ -586,3 +590,8 @@ func logVerificationResult(logger log.Logger, result *notation.ValidationResult)
 		logger.Errorf("%v validation failed. Failure reason: %v", result.Type, result.Error)
 	}
 }
+
+func versionCompare(minVer string, pluginVer string) int{
+    pluginVer = "v" + pluginVer
+    return semver.Compare(minVer, pluginVer)
```

Comment:
do we need to add `v` prefix for minVer ?

Reply:
If the version emitted from signature doesn't have V as suffix, then Yes

Reply:
plugin should emit a version that is a valid semver but we still need to validate that. Also, plugin should never emit a semver that is prefixed with `v` as its an invalid semver.

 
https://github.com/notaryproject/notaryproject/blob/c4400ebb2da98bce89e66cc8fadeddae564e2b61/specs/plugin-extensibility.md?plain=1#L444

Reply:
Added the plugin version validation code